report/sections/secondary-groups.tex

```diff
@@ -5,14 +5,14 @@ \section{Secondary Groups}
 
 \subsection{Computer Vision Group}
 %Format read from CV Group
-After training our models with the Bosphorus Database we need to predict an emotion by reading input from the CV group. They are creating an array containing $25$ landmarks and passing it as input into our models. PCA was used for dimensionality reduction before training our model, so the same PCA model is used on the given input for reducing the number of features.
```

Comment:
I felt like mentioning the Bosphorus Database over and over was redundant, they already know what dataset we're using